kpops/components/base_components/kafka_connector.py

```diff
@@ -98,7 +100,16 @@ def helm(self) -> Helm:
         )
         return helm
 
-    def _get_resetter_helm_chart(self) -> str:
+    @property
+    def _connector_resetter_release_name(self) -> str:
+        """Get connector resetter's release name"""
+        suffix = "-clean"
+        clean_up_release_name = self.full_name + suffix
+        trimmed_name = trim_release_name(clean_up_release_name, suffix)
+        return trimmed_name
+
+    @property
+    def _resetter_helm_chart(self) -> str:
```

Comment:
what is the difference between `_resetter_helm_chart` and `_kafka_connect_resetter_chart`?

Reply:
They are the same. I removed one

Reply:
I still see both in the diff?

Reply:
Just pushed